pkg/logql/log/fmt.go

```diff
@@ -246,21 +260,29 @@ func NewLabelsFormatter(fmts []LabelFmt) (*LabelsFormatter, error) {
 	}
 	formats := make([]labelFormatter, 0, len(fmts))
```

Comment:
nit: it can be len(fmts)+2 now?

Reply:
I don't think this changes? I'm adding to the function map but not the formats slice. 

The functions list does get incremented by 2 to add the 2 new functions

https://github.com/grafana/loki/blob/286dbfbde38a035a4f23f2b41b1e5b658a0650e0/pkg/logql/log/fmt.go#L94